cas/ALV.md

```diff
@@ -57,7 +58,24 @@ Acceptable remediation:
     * If the certificate has been in existence for past audit periods, then you must also file an [Incident Report](https://www.ccadb.org/cas/incident-report).
     * An Incident Report may not be needed if the certificate is self-signed and has the same Subject + SPKI as other certificates listed in the audit statement. For example, this can happen when a Root Store includes one version of a root certificate, but another version of the root certificate can be part of a valid chain constructed as: leaf --> untrusted root --> trusted root.
 * Revoke the intermediate certificate in accordance with Root Store policies and the BRs.
-    * If your CA decides not to revoke the certificate within the timeline specified by section 4.9 of the BRs, then that is another incident, which must be addressed in a separate [Incident Report](https://www.ccadb.org/cas/incident-report). 
+    * If your CA decides not to revoke the certificate within the timeline specified by section 4.9 of the BRs, then that is another incident, which must be addressed in a separate [Incident Report](https://www.ccadb.org/cas/incident-report).
+ 
+## Testing Audit Statements with ALV ##
+CA Owners may want to test preliminary audit statements to ensure the file will pass ALV before collecting a final copy from their auditor. To test preliminary audit statements a CA Owner should:
+
+1. Create the [Add/Update Root Request Case](https://www.ccadb.org/cas/updates) in the CCADB.
+2. In the ‘AUDITS’ tab of the Case, add [Audit Firm](https://docs.google.com/document/d/12U4az-hjYDC_aWsVn8-Y5vVmJ10inVziAxrQoxP-hfI/edit#heading=h.y0j6nqw48aqv) information.
+3. Add [Audit Information](https://docs.google.com/document/d/12U4az-hjYDC_aWsVn8-Y5vVmJ10inVziAxrQoxP-hfI/edit#heading=h.fo56qxxtqyjb) to the Case, including a URL to where the preliminary audit statement is located.
+    * These preliminary statements are typically [uploaded](https://www.ccadb.org/cas/fields#uploading-documents) to Bugzilla.
+4. Add a Case Comment to the ‘CASE PROGRESS’ tab of the Case that says: “Preliminary Audit Reports” to inform Root Store Operators of your intent to test the preliminary audit statements.
+5. [Run ALV](https://docs.google.com/document/d/12U4az-hjYDC_aWsVn8-Y5vVmJ10inVziAxrQoxP-hfI/edit#heading=h.risbuur7q7a).
+    * Ignore any ALV error related to the audit report not originating from a trusted location.
+    * Work with the auditor to resolve other [common ALV findings](https://www.ccadb.org/cas/alv#common-alv-findings) and errors.
```

Comment:
```suggestion
    * Work with the auditor to resolve other [common ALV findings](https://www.ccadb.org/cas/alv#common-alv-findings) and errors.
    * If you encounter a problem not described on the [common ALV findings](https://www.ccadb.org/cas/alv#common-alv-findings) page, contact support[at]ccadb[dot]org.
```

Nit: Clarify where CAs should seek assistance for uncommon errors (i.e., discourage emailing individuals, given we work as a team to process cases via the Steering Committee rotation).